README.md

```diff
@@ -2,24 +2,45 @@
 
 This scans every ticker on the market, gets their last 5 months of volume history, and alerts you when a stock's volume exceeds 10 standard deviations from the mean within the last 3 days. (these numbers are all adjustable).  Helps find anomalies in the stock market
 
+## Requirements
 
-## How to run this:
+Docker - install Docker https://docs.docker.com/get-docker/
 
--download your favorite Python IDE. (I use VSCode)
+Alternatively,
 
--get my script from GitHub
+1) download your favorite Python IDE. (For example, the project creator uses VSCode)
 
--open the script in your IDE and install all required dependancies by typing pip install -r requirements.txt into the IDE's terminal. You can get to the the terminal on VSC by pressing CMD and ` at the same time.
+2) Clone project from GitHub
 
--run the market_scanner.py and it will print out results into the terminal
--you can also graph any ticker's volume in grapher.py
+3) Open project in your IDE and install all required dependancies by typing pip install -r requirements.txt into the IDE's terminal. You can get to the the terminal on VSC by pressing CMD and ` at the same time.
+
+## How to run
+Build docker image by running
```

Comment:
you may want to explain how to install docker, or even link to the install page here: 
https://docs.docker.com/get-docker/

Reply:
That's already noted in the requirements section, line #7 of the readme.

Reply:
@BeauBouchard See line 7 and @RKosow it's very obvious but also a good idea, adding that now